### PR TITLE
add sampleqc paper

### PR DESCRIPTION
**Pull request by markrobinsonuzh**

### CODE REVIEW

**Review by supermaxiste — CHANGES_REQUESTED** [review]
Link to SDSC to fix and typo on my last name. Minor: Will's full name not in bold in the html. Rest looks good.

**Comment by markrobinsonuzh** [comment]
@supermaxiste Sharp eyes. Thanks! Should be all fixed now.

File: people.Rmd [hunk 1]
```diff
@@ -563,6 +563,7 @@ for (i in seq_along(tables_all)) {
 ```
 
 ## Alumni
+- Stefan Milosajevic (PhD student shared with Shimizu group; author of [ARPEGGIO](https://github.com/supermaxiste/ARPEGGIO); now Biomedical Data Engineer at [https://datascience.ch](Swiss Data Science Center))
```
[the remark this answers is not in the document]
Comment (markrobinsonuzh) [reply people.Rmd]: fixed.

[the remark this answers is not in the document]
Comment (markrobinsonuzh) [reply people.Rmd]: fixed. sorry!

File: publications.html [hunk 2]
```diff
@@ -500,6 +486,28 @@ <h2>Preprints</h2>
 </ul>
 </div>
 <div id="section" class="section level2">
+<h2>2023</h2>
+<ul>
+<li>Macnair W, <strong>Robinson M</strong>: SampleQC: robust
```
[the remark this answers is not in the document]
Comment (markrobinsonuzh) [reply publications.html]: done.

**Review by supermaxiste — APPROVED** [review]

**Pull request merged**